dgm-builder/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>com.cloudbees</groupId>
         <artifactId>groovy-cps-parent</artifactId>
-        <version>1.21-SNAPSHOT</version>
+        <version>1.21-stop-recursion2-SNAPSHOT</version>
```

Comment:
Revert please. Not necessary for testing downstream anyway; just use the timestamp in the dependency.

Reply:
It'll be reverted before merge -- I'm using this for local testing and don't seem to have the capability to deploy a SNAPSHOT -- and this isn't part of the repository-permissions-updater.